### PR TITLE
UI tests - Update matrice to run sanity tests on last version of each minor

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update matrice to run sanity tests on last version of each minor
| Type?             | refactor
| Sponsor company   | @PrestaShopCorp
| How to test?      | CI is :green_apple:

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Tests may be failing, but I'm merging anyway as we absolutely need to reduce the number of tests to run at each push / pull request.

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
